package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weather-app-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "jest",
+    "start": "node dist/server.js",
+    "build": "tsc"
```

Comment:
-test with covarage
- sonar code
- how to run linting 
- how to deploy on local, test, prod?